pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/GetResult.java

```diff
@@ -26,11 +26,20 @@
 @Data
 public class GetResult {
 
+    public GetResult(byte[] value, Stat stat){
+        this.value = value.clone();
```

Comment:
While I understand the immutability aspect, making 2 extra copies and allocations of the value array it's overkill for this scenario.

Reply:
Maybe we could make the `value` public to solve this problem(`EI_EXPOSE_REP`) and avoid copy, even though that would sacrifice immutability. What do you think?

Reply:
I'm not super convinced in making the field public. `GetResult` class is part of API so I'd rather keep that behind a getter method. An implementation might not store it directly as a `byte[]` but instead convert that on demand.

Reply:
Can we instead ignore the warning for this case?

Reply:
Okay.